lighthouse-core/lib/log.js

```diff
@@ -17,20 +17,27 @@
 'use strict';
 
 const debug = require('debug');
-const debugNode = require('debug/node');
 const EventEmitter = require('events').EventEmitter;
 
+const isBrowser = process.browser;
```

Comment:
maybe a quick comment to mention this is created by browserify? I was wondering what on earth this was for a minute :)